api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -225,11 +228,10 @@ private void startMetricsService() {
             metricsService.setDateTimeService(createDateTimeService());
             metricsService.setDefaultTTL(getDefaultTTL());
 
-            // TODO Set up a managed metric registry
-            // We want a managed registry that can be shared by the JAX-RS endpoint and the core. Then we can expose
-            // the registered metrics in various ways such as new REST endpoints, JMX, or via different
-            // com.codahale.metrics.Reporter instances.
-            metricsService.startUp(session, keyspace, false, false, new MetricRegistry());
+            MetricRegistry metricRegistry = MetricRegistryProvider.INSTANCE.getMetricRegistry();
+            jmxReporter = JmxReporter.forRegistry(metricRegistry).inDomain("hawkular.metrics").build();
```

Comment:
I'm late to the game on this PR, but I tend to agree with Stefan. Unless we need to always have JMXReporter running, I would prefer to keep it off or at least be able to turn it off. This made me think of a bug with an early version of the DataStax driver that involved JMXReporter that we hit in RHQ. It may have been a thread leak. I do not recall the specifics. What I do recall though is that disabling the reporter worked around the issue.